source/info-tool/ev3dev-system-info.sh

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env bash
 
-if [ "$1" == '--help' ]; then
+FORMAT_MARKDOWN=false
+if [ "$1" = "--help" ]; then
```

Comment:
This would probably be better written as a case statement.

    case $1 in
    -h|--help)
        do_stuff
        ;;
    -m|--markdown)
        do_stuff
        ;;
    esac